test/bzz-protocol.spec.ts

```diff
@@ -130,9 +157,8 @@ describe('BZZ protocol', () => {
     }
     await changeUrl(testUrlValue)
     //test whether it had affect on routing
-    const bzzPage = await global.__BROWSER__.newPage()
-    const errorRegExp = /^net::ERR_CONNECTION_REFUSED/
-    await expect(bzzPage.goto(bzzReferenceByGoogle('nevermind-value'))).rejects.toThrowError(errorRegExp)
+    const bzzPage = await newBzzpage(bzzReferenceByGoogle('nevermind-value'))
+    expect(bzzPage.url()).toBe('chrome-error://chromewebdata/')
```

Comment:
Maybe add a comment here that explains that `chrome-error://chromewebdata/` means it was redirected to the proper port, and this is the expected error page.